tests/containers/rootless_podman.pm

```diff
@@ -100,10 +102,17 @@ sub run {
     $podman->cleanup_system_host();
 }
 
-sub softfail_and_skip_on_bsc1182874 {
-    my $alpine = registry_url('alpine');
-    if (script_run("podman run $alpine", timeout => 180) != 0) {
-        record_soft_failure "bsc#1182874 - container fails to run in rootless mode";
+sub softfail_on_bsc1197093 {
+    # By default the storage driver is set to btrfs if /var is in btrfs
+    # but if the home partition is not btrfs podman commands will fail with
+    # Error: "/home/bernhard/.local/share/containers/storage/btrfs" is not on a btrfs filesystem
+    if (script_output("podman info", proceed_on_failure => 1) =~ m/prerequisites for driver not satisfied/) {
+        record_soft_failure("bsc#1197093 - /home partition is in different filesystem");
+        record_info('partitions', script_output('lsblk -f'));
+        record_info('storage.conf', script_output('cat /etc/containers/storage.conf'));
+        assert_script_run('sed \'s/^driver.*/driver \= \"overlay\"/\' -i /etc/containers/storage.conf');
+        assert_script_run("rm -rf /home/$user/.local/share/containers/");
+        assert_script_run('podman info');
         return 1;
```

Comment:
Should the test continue after it changes the driver, or it should exit anyway ?

Reply:
ah the return on L#96 is gone, please the returns in `softfail_on_bsc1197093` as they are meaningless now.

Reply:
yes, done :)